### PR TITLE
Update ict baseline pages

### PR DESCRIPTION
**Pull request by alexwilsongsa**

### CODE REVIEW

File: _pages/acquisition/2018-05-29-buy.md [hunk 1]
```diff
@@ -262,6 +262,10 @@ Trust but verify: before awarding a contract or deploying custom-developed deliv
   <p>
     For organizations that test for Section 508 conformance following practices other than the Trusted Tester protocol, the toolset and approaches utilized must be aligned with the <a href="https://section508coordinators.github.io/ICTTestingBaseline/" target="_blank">ICT Testing Baseline</a> to ensure consistency in testing practices across agencies.
   </p>
+
+  <p>
+    The ICT Testing Baseline helps ensure that web content is accessible to people with disabilities and meets the legal obligations of Section 508. The ICT Testing Baseline describes how to evaluate conformance to <a href="https://www.access-board.gov/ict/">the Revised 508 Standards</a>, which reference and align with international standards such as WCAG 2.0 Level A and AA Success Criteria. It is organized by categories to help you easily identify applicable requirements. Note that the Baseline is not a test process and does not include testing tools; rather, it should be used to create an accessibility test process.
```
Comment (kengdoj) [inline _pages/acquisition/2018-05-29-buy.md:267]:
```suggestion
    The ICT Testing Baseline for Web helps ensure that web content is accessible to people with disabilities and meets the technical requirements of Section 508. The ICT Testing Baseline describes how to evaluate conformance to <a href="https://www.access-board.gov/ict/">the Revised 508 Standards</a>, which incorporate by reference the Web Content Accessibility Guidelines (WCAG) 2.0 Level A and AA Success Criteria. The Baseline for Web is organized by categories to help you easily identify applicable requirements. Note that the Baseline is not a test process and does not include testing tools; rather, it should be used to create an accessibility test process or validate an existing test process for completeness.
```

File: _pages/test/2018-05-22-test-web-software.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ created: 1527026399
...
 
+## ICT Testing Baseline
+
+The [ICT Testing Baseline][20] establishes the minimum standards for web accessibility in the federal government. It supports the use of automated and manual testing tools to ensure a comprehensive and accurate assessment of web accessibility. Results from a baseline-aligned test process provide the basis for determining if web content meets the applicable technical requirements for web accessibility. A test process that aligns with the ICT Testing Baseline can reliably determine if web content meets these requirements. The ICT Testing Baseline also supports harmonization and interoperability of test results and tools, and sharing of best practices and resources among testers and developers. 
+
+The ICT Testing Baseline describes how to evaluate conformance to [the Revised 508 Standards][21], which align with WCAG 2.0 Level A and AA Success Criteria. It is organized by categories to help you easily identify applicable requirements. Note that the Baseline is not a test process and does not include testing tools; rather, it should be used to ***create*** an accessibility test process.
```
Comment (kengdoj) [inline _pages/test/2018-05-22-test-web-software.md:16]:
```suggestion
The ICT Testing Baseline describes what to evaluate for conformance to [the Revised 508 Standards][21], which align with WCAG 2.0 Level A and AA Success Criteria. It is organized by categories to help you easily identify applicable requirements. Note that the Baseline is not a test process and does not include testing tools; rather, it should be used to ***create*** an accessibility test process or validate an existing test process for completeness.
```

**Review by kengdoj — CHANGES_REQUESTED** [review]
A few suggested edits. Thanks!

**Comment by drewnielson** [comment]
> A few suggested edits. Thanks!

Thanks @kengdoj. Suggested changes incorporated, and added USAB as contributor.

**Review by drewnielson — APPROVED** [review]

**Pull request merged**